site/content/docs/4.3/components/navbar.md

```diff
@@ -245,29 +259,25 @@ Input groups work, too:
 
 {{< example >}}
 <nav class="navbar navbar-light bg-light">
-  <div class="container-fluid">
-    <form class="form-inline">
-      <div class="input-group">
-        <div class="input-group-prepend">
-          <span class="input-group-text" id="basic-addon1">@</span>
-        </div>
-        <input type="text" class="form-control" placeholder="Username" aria-label="Username" aria-describedby="basic-addon1">
+  <form class="container-fluid">
+    <div class="input-group">
+      <div class="input-group-prepend">
+        <span class="input-group-text" id="basic-addon1">@</span>
       </div>
-    </form>
-  </div>
+      <input type="text" class="form-control" placeholder="Username" aria-label="Username" aria-describedby="basic-addon1">
+    </div>
+  </form>
 </nav>
 {{< /example >}}
 
 Various buttons are supported as part of these navbar forms, too. This is also a great reminder that vertical alignment utilities can be used to align different sized elements.
```

Comment:
We should call out that if your navbar is an entire form, or mostly form, you can use the `<form>` element as the container and save some HTML. Applies to the option above and below this copy. Otherwise this catches me off guard.